core/src/pixelcolor/binary_color.rs

```diff
@@ -83,8 +83,11 @@ impl BinaryColor {
     /// assert!(BinaryColor::On.is_on());
     /// ```
     #[inline]
-    pub fn is_on(self) -> bool {
-        self == BinaryColor::On
+    pub const fn is_on(self) -> bool {
+        match self {
```

Comment:
`==` can't be used in a `const` function on `BinaryColor`, so I switched this to a `match`.

Reply:
This could also be written as: `matches!(self, BinaryColor::On)`